### ISSUE TITLE
Displaying 0 degrees instead of correct value

### ISSUE THREAD

**Issue by kamraten** [danirabbit/nimbus#34]
Degrees shown is, and has been since yesterday, showing 0 instead of the correct value.
Location used is Gothenburg, Sweden.
Screenshot can be found [here](https://i.imgur.com/WiHvGyA.png).

**Comment by jkdele** [comment]
Have you tried running Nimbus through the terminal? I assume we're having the same issue ([#30](https://github.com/danrabbit/nimbus/issues/30)).

**Comment by kamraten** [comment]
I have not tried running it trough the terminal yet, i dont know how. I typed `nimbus` and `Nimbus` but it returns command not found,

**Comment by jkdele** [comment]
Try `com.github.danrabbit.nimbus`

**Comment by kamraten** [comment]
Typing `com.github.danrabbit.nimbus` in the terminal gives me this:
```
(com.github.danrabbit.nimbus:3484): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2045:14: 'min-height' is not a valid property name

(com.github.danrabbit.nimbus:3484): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2046:13: 'min-width' is not a valid property name

(com.github.danrabbit.nimbus:3484): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:2051:13: 'min-width' is not a valid property name
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

(com.github.danrabbit.nimbus:3484): GWeather-WARNING **: Failed to get IWIN forecast data: 403 Forbidden
```
And the same window as my screenshot, 0 degrees and sunny. I have no themes installed, just elementary and apps from their store.

**Comment by valkirilov** [comment]
I have the same issue on a clean installation of elementaryOS. Nimbus displayed the correct values when installed, but after this I see only 0 degrees.

Here is my output:
```
(com.github.danrabbit.nimbus:3805): GWeather-WARNING **: Failed to get METAR data: 404 Not Found.
(com.github.danrabbit.nimbus:3805): GWeather-WARNING **: Failed to get IWIN forecast data: 403 Forbidden
(com.github.danrabbit.nimbus:3805): GWeather-WARNING **: Failed to get METAR data: 404 Not Found.
```

**Comment by danirabbit** [comment]
Hey thanks for your report! This was probably fixed by #133
